tests/wpt/runner/expectation.json

```diff
@@ -9789,7 +9789,7 @@
     "event_constructor.window.html": false,
     "event_initstorageevent.window.html": false,
     "missing_arguments.window.html": true,
-    "storage_builtins.window.html": true,
+    "storage_builtins.window.html": false,
```

Comment:
This test checks if
```
Storage.prototype.setItem == localStorage.setItem
```

but our Proxy needs to create a new instance via `.bind` because receiver holds cppgc object.